alignak/objects/item.py

```diff
@@ -155,6 +155,7 @@ def __init__(self, params=None, parsing=True):
                               "it is not in %s object properties" % \
                               (key, cls.__name__)
                     self.configuration_warnings.append(warning)
+                    self.properties[key] = ToGuessProp(default='')
```

Comment:
The properties defined in modules (additional modules), are not pass after serialization because they not have property type (so it's guessprop) and the module not have these properties if this line not added